pow/src/main/java/tech/pegasys/teku/pow/Eth1HeadTracker.java

```diff
@@ -82,6 +90,10 @@ private void onLatestBlockHead(final Block headBlock) {
     if (headAtFollowDistance
         .map(current -> current.compareTo(newHeadAtFollowDistance) < 0)
         .orElse(true)) {
+      if (!reachedHead) {
```

Comment:
`reachedHead` will need to be an `AtomicBoolean` as multiple threads could be involved.